### DISCUSSION TITLE
React native and Chakra-ui?

### DISCUSSION THREAD

**Discussion by efegulec1** [chakra-ui/chakra-ui#5390] [General]
Hello all,

When will Chakra-ui be available for React Native? We're about to start a project for a client of ours (an ecommerce giant in Australia) -- and we'd like to use Chakra-ui instead of nativebase

**Comment by dionrussell** [comment]
Any updates on this?

**Comment by merganon** [comment]
Check the discussion here: https://github.com/chakra-ui/chakra-ui/issues/183
